src/libs/MoneyRequestReportUtils.ts

```diff
@@ -88,10 +88,6 @@ function isSingleTransactionReport(report: OnyxEntry<Report>, transactions: Tran
  * Likewise, if report has only 1 connected transaction, then we also use other views.
  */
 function shouldDisplayReportTableView(report: OnyxEntry<Report>, transactions: Transaction[]) {
-    if (!report || !transactions.length) {
-        return false;
-    }
-
     return !isReportTransactionThread(report) && !isSingleTransactionReport(report, transactions);
```

Comment:
@martasudol what do you think about this? Maybe this could work 🤔 
```
if (!report) {
     return false;
}

return (isMoneyRequestReport(report) || isInvoiceReport(report)) && !isReportTransactionThread(report) && !isSingleTransactionReport(report, transactions);
```